zulip_botserver/zulip_botserver/server.py

```diff
@@ -201,7 +201,7 @@ def handle_bot() -> str:
     bot_handler = app.config.get("BOT_HANDLERS", {})[bot]
     message_handler = app.config.get("MESSAGE_HANDLERS", {})[bot]
     is_mentioned = event["trigger"] == "mention"
-    is_private_message = event["trigger"] == "private_message"
+    is_direct_message = event["trigger"] in ["direct_message", "private_message"]
```

Comment:
Should we add a `TODO/compatibility` comment here to remove `private_message` in future? I don't see it included in
[Zulip PyPI packages release checklist](https://zulip.readthedocs.io/en/latest/subsystems/api-release-checklist.html)

Reply:
Yeah I think it would be valuable to put that kind of comment here to have a record why this is needed and to remove it at some point

Reply:
Added a comment:
```
    # TODO/compatibility: Remove the support for "private_message" as a valid
    # trigger value when one can no longer directly upgrade from 0.8.x to main.
```